xwiki-platform-tools/xwiki-platform-tool-packager-plugin/pom.xml

```diff
@@ -335,6 +335,13 @@
       <version>${project.version}</version>
       <scope>runtime</scope>
     </dependency>
+    <!-- URL Default Module -->
+    <dependency>
+      <groupId>org.xwiki.platform</groupId>
+      <artifactId>xwiki-platform-url-default</artifactId>
+      <version>${project.version}</version>
+      <scope>runtime</scope>
+    </dependency>
```

Comment:
Maybe check the minimal distribution. It has:

```
    <!-- URL -->
    <dependency>
      <groupId>org.xwiki.platform</groupId>
      <artifactId>xwiki-platform-url-scheme-standard</artifactId>
      <version>${project.version}</version>
      <scope>runtime</scope>
    </dependency>
```

I think `xwiki-platform-url-scheme-standard` should now have a dep on the url impl instead of the url api.

Reply:
> I think xwiki-platform-url-scheme-standard should now have a dep on the url impl instead of the url api.

Not sure, if the purpose is just to avoid adding a dep in the minimal distrib. In terms of archi, it feels better to have a url API used by both url default and url-scheme-standard: the purpose of both modules is specific.